---a/net/minecraft/scoreboard/IScoreObjectiveCriteria.java
+++b/net/minecraft/scoreboard/IScoreObjectiveCriteria.java
@@ -1,11 +1,10 @@
 package net.minecraft.scoreboard;
 
-import com.google.common.collect.Maps;
 import java.util.List;
 import java.util.Map;
+
 import net.minecraft.util.EnumChatFormatting;
 
-public interface IScoreObjectiveCriteria
-{
+import com.google.common.collect.Maps;
+
+public interface IScoreObjectiveCriteria {
+
     Map INSTANCES = Maps.newHashMap();
     IScoreObjectiveCriteria DUMMY = new ScoreDummyCriteria("dummy");
     IScoreObjectiveCriteria TRIGGER = new ScoreDummyCriteria("trigger");
@@ -14,8 +14,42 @@
     IScoreObjectiveCriteria playerKillCount = new ScoreDummyCriteria("playerKillCount");
     IScoreObjectiveCriteria totalKillCount = new ScoreDummyCriteria("totalKillCount");
     IScoreObjectiveCriteria health = new ScoreHealthCriteria("health");
-    IScoreObjectiveCriteria[] field_178792_h = new IScoreObjectiveCriteria[] {new GoalColor("teamkill.", EnumChatFormatting.BLACK), new GoalColor("teamkill.", EnumChatFormatting.DARK_BLUE), new GoalColor("teamkill.", EnumChatFormatting.DARK_GREEN), new GoalColor("teamkill.", EnumChatFormatting.DARK_AQUA), new GoalColor("teamkill.", EnumChatFormatting.DARK_RED), new GoalColor("teamkill.", EnumChatFormatting.DARK_PURPLE), new GoalColor("teamkill.", EnumChatFormatting.GOLD), new GoalColor("teamkill.", EnumChatFormatting.GRAY), new GoalColor("teamkill.", EnumChatFormatting.DARK_GRAY), new GoalColor("teamkill.", EnumChatFormatting.BLUE), new GoalColor("teamkill.", EnumChatFormatting.GREEN), new GoalColor("teamkill.", EnumChatFormatting.AQUA), new GoalColor("teamkill.", EnumChatFormatting.RED), new GoalColor("teamkill.", EnumChatFormatting.LIGHT_PURPLE), new GoalColor("teamkill.", EnumChatFormatting.YELLOW), new GoalColor("teamkill.", EnumChatFormatting.WHITE)};
-    IScoreObjectiveCriteria[] field_178793_i = new IScoreObjectiveCriteria[] {new GoalColor("killedByTeam.", EnumChatFormatting.BLACK), new GoalColor("killedByTeam.", EnumChatFormatting.DARK_BLUE), new GoalColor("killedByTeam.", EnumChatFormatting.DARK_GREEN), new GoalColor("killedByTeam.", EnumChatFormatting.DARK_AQUA), new GoalColor("killedByTeam.", EnumChatFormatting.DARK_RED), new GoalColor("killedByTeam.", EnumChatFormatting.DARK_PURPLE), new GoalColor("killedByTeam.", EnumChatFormatting.GOLD), new GoalColor("killedByTeam.", EnumChatFormatting.GRAY), new GoalColor("killedByTeam.", EnumChatFormatting.DARK_GRAY), new GoalColor("killedByTeam.", EnumChatFormatting.BLUE), new GoalColor("killedByTeam.", EnumChatFormatting.GREEN), new GoalColor("killedByTeam.", EnumChatFormatting.AQUA), new GoalColor("killedByTeam.", EnumChatFormatting.RED), new GoalColor("killedByTeam.", EnumChatFormatting.LIGHT_PURPLE), new GoalColor("killedByTeam.", EnumChatFormatting.YELLOW), new GoalColor("killedByTeam.", EnumChatFormatting.WHITE)};
+    IScoreObjectiveCriteria[] field_178792_h = new IScoreObjectiveCriteria[]{
+            new GoalColor("teamkill.", EnumChatFormatting.BLACK),
+            new GoalColor("teamkill.", EnumChatFormatting.DARK_BLUE),
+            new GoalColor("teamkill.", EnumChatFormatting.DARK_GREEN),
+            new GoalColor("teamkill.", EnumChatFormatting.DARK_AQUA),
+            new GoalColor("teamkill.", EnumChatFormatting.DARK_RED),
+            new GoalColor("teamkill.", EnumChatFormatting.DARK_PURPLE),
+            new GoalColor("teamkill.", EnumChatFormatting.GOLD),
+            new GoalColor("teamkill.", EnumChatFormatting.GRAY),
+            new GoalColor("teamkill.", EnumChatFormatting.DARK_GRAY),
+            new GoalColor("teamkill.", EnumChatFormatting.BLUE),
+            new GoalColor("teamkill.", EnumChatFormatting.GREEN),
+            new GoalColor("teamkill.", EnumChatFormatting.AQUA),
+            new GoalColor("teamkill.", EnumChatFormatting.RED),
+            new GoalColor("teamkill.", EnumChatFormatting.LIGHT_PURPLE),
+            new GoalColor("teamkill.", EnumChatFormatting.YELLOW),
+            new GoalColor("teamkill.", EnumChatFormatting.WHITE)
+    };
+    IScoreObjectiveCriteria[] field_178793_i = new IScoreObjectiveCriteria[]{
+            new GoalColor("killedByTeam.", EnumChatFormatting.BLACK),
+            new GoalColor("killedByTeam.", EnumChatFormatting.DARK_BLUE),
+            new GoalColor("killedByTeam.", EnumChatFormatting.DARK_GREEN),
+            new GoalColor("killedByTeam.", EnumChatFormatting.DARK_AQUA),
+            new GoalColor("killedByTeam.", EnumChatFormatting.DARK_RED),
+            new GoalColor("killedByTeam.", EnumChatFormatting.DARK_PURPLE),
+            new GoalColor("killedByTeam.", EnumChatFormatting.GOLD),
+            new GoalColor("killedByTeam.", EnumChatFormatting.GRAY),
+            new GoalColor("killedByTeam.", EnumChatFormatting.DARK_GRAY),
+            new GoalColor("killedByTeam.", EnumChatFormatting.BLUE),
+            new GoalColor("killedByTeam.", EnumChatFormatting.GREEN),
+            new GoalColor("killedByTeam.", EnumChatFormatting.AQUA),
+            new GoalColor("killedByTeam.", EnumChatFormatting.RED),
+            new GoalColor("killedByTeam.", EnumChatFormatting.LIGHT_PURPLE),
+            new GoalColor("killedByTeam.", EnumChatFormatting.YELLOW),
+            new GoalColor("killedByTeam.", EnumChatFormatting.WHITE)
+    };
 
     String getName();
 
@@ -25,8 +25,7 @@
 
     IScoreObjectiveCriteria.EnumRenderType getRenderType();
 
-    public static enum EnumRenderType
-    {
+    public static enum EnumRenderType {
         INTEGER("integer"),
         HEARTS("hearts");
         private static final Map field_178801_c = Maps.newHashMap();
@@ -34,33 +34,27 @@
 
         private static final String __OBFID = "CL_00001960";
 
-        private EnumRenderType(String p_i45548_3_)
-        {
+        private EnumRenderType(String p_i45548_3_) {
             this.field_178798_d = p_i45548_3_;
         }
 
-        public String func_178796_a()
-        {
+        public String func_178796_a() {
             return this.field_178798_d;
         }
 
-        public static IScoreObjectiveCriteria.EnumRenderType func_178795_a(String p_178795_0_)
-        {
-            IScoreObjectiveCriteria.EnumRenderType enumrendertype = (IScoreObjectiveCriteria.EnumRenderType)field_178801_c.get(p_178795_0_);
+        public static IScoreObjectiveCriteria.EnumRenderType func_178795_a(String p_178795_0_) {
+            IScoreObjectiveCriteria.EnumRenderType enumrendertype = (IScoreObjectiveCriteria.EnumRenderType) field_178801_c.get(p_178795_0_);
             return enumrendertype == null ? INTEGER : enumrendertype;
         }
 
-        static
-        {
+        static {
             IScoreObjectiveCriteria.EnumRenderType[] var0 = values();
             int var1 = var0.length;
 
-            for (int var2 = 0; var2 < var1; ++var2)
-            {
+            for (int var2 = 0; var2 < var1; ++var2) {
                 IScoreObjectiveCriteria.EnumRenderType var3 = var0[var2];
                 field_178801_c.put(var3.func_178796_a(), var3);
             }
         }
     }
-}
+}
